tests/test_arena.py

```diff
@@ -1,5 +1,36 @@
 import arena
+from unittest import TestCase
 
 
-def test_dummy():
-    assert arena.WORKS
+class TestDamage(TestCase):
+
+    def test_damage_parsing_success(self):
+        damage1 = arena.Damage("1d2")
+        self.assertTrue(damage1._Damage__x == 1)
+        self.assertTrue(damage1._Damage__y == 2)
+        self.assertTrue(damage1._Damage__z == 0)
+
+        damage2 = arena.Damage("1d2 + 3")
+        self.assertTrue(damage2._Damage__x == 1)
+        self.assertTrue(damage2._Damage__y == 2)
+        self.assertTrue(damage2._Damage__z == 3)
+        pass
+
+    def test_damage_drawing(self):
+        damage = arena.Damage("1d2 + 3")
+        self.assertTrue(damage.draw() > 0)
```

Comment:
If you want to test code using random number generator you need to either [mock](https://stackoverflow.com/questions/39930244/how-to-mock-random-choice-in-python) the random.randint function or move it as a dependency to your class.